help/ru_RU/success.yml

```diff
@@ -29,5 +29,6 @@ blockbuster.success:
         clean: "{f}В записи игрока %s{f} свойства %s{f} были успешно очищены от %s{f} к %s{f}!"
         prolonged: "{f}Запись игрока %s{f} была продлена. Теперь задержки записи составляют: %s{f} до и %s{f} после соответственно."
         faded: "{f}В запись игрока %s{f}на кадры %s{f} был наложен переход."
+        flipped: "{f}Запись игрока %s{f} была успешно повёрнута по оси %s{f} в координате %s{f} !"
         cut: "{f}Из записи игрока %s{f} были вырезаны кадры между %s{f} и %s{f} ."
         restored: "{f}У записи игрока %s{f} итерация %s{f} была восстановлена!"
```

Comment:
I forgot to add `/record reverse`'s [success message](https://github.com/mchorse/blockbuster/compare/1.6.2...master#diff-b01930136fd9ef471a252d9e462a0036R35), could you add it?